web/src/app/monitors/monitors-list/monitors-list.component.ts

```diff
@@ -29,9 +29,9 @@ export class MonitorsListComponent implements OnInit {
    * Lifecycle init method
    */
   ngOnInit(): void {
-    this.uid = this.route.snapshot.paramMap.get('projectUid');
+    this.projectUid = this.route.snapshot.paramMap.get('projectUid');
     this.projectSubscription = this.projectService
-      .findOneById(this.uid)
+      .findOneById(this.projectUid)
```

Comment:
This would be better as a resolve, so the page does not load empty initially. I like the way `tokens-list` is done https://github.com/DashboardHub/PipelineDashboard/blob/v0.11.6/web/src/app/tokens/tokens-list/tokens-list.component.ts#L27

Reply:
done

Reply:
Thank you